src/Rive.tsx

```diff
@@ -124,6 +129,7 @@ const RiveContainer = React.forwardRef<RiveRef, Props>(
       animationName,
       stateMachineName,
       testID,
+      disableDefaultTouchHandling = false,
```

Comment:
Can reduce this variable and do something like this
`const isDisableDefaultTouch = onPress`

Reply:
can ignore this if we need to support `onPressIn` at the same time too.